File: bddtests/src/main/kotlin/mocking/vision/models/PrescriptionHistory.kt
package mocking.vision.models

import javax.xml.bind.annotation.XmlElement
import javax.xml.bind.annotation.XmlRootElement

@XmlRootElement(name = "prescriptionHistory")
data class PrescriptionHistory(

    @XmlElement(namespace= "urn:vision", name = "request")
    var request: MutableList<Request> = mutableListOf()
)

File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Vision/Models/Prescriptions/PrescriptionHistory.cs
using System.Collections.Generic;
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Vision.Models.Prescriptions
{
    public class PrescriptionHistory
    {
        [XmlElement(ElementName = "request", Namespace = "urn:vision")]
        public List<Request> Requests { get; set; } = new List<Request>();
    }
}
